index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  </head>
+<style>
+    @media all {
+        #features ul,
+        #nav-bar {
+            display: flex;
+            flex-direction: row;
+            align-content: space-between;
+            align-items: center;
+        }
+        h1 {
+          margin-top: 40px;
+        }
+        header {
+            max-width: 60%;
+            margin: auto;
+        }
+        #nav-bar {
+          position: fixed;
+          top: 0;
+          background: black;
+          color: white;
+          width: 100%;
+        }
+
+        a {
+          color: white;
+        }
+        ul {
+          list-style-type: none;
+        }
+        li {
+          margin-right: 30px;
+        }
+        #nav-bar a, ul li {
+          align-self: center;
+        }
+        
+    }
+</style>
```

Comment:
Please move these styles to an external CSS stylesheet as it's a best practice to have a separation of concerns.